src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
+    private Color[] colors = Color.values();
```

Comment:
The array 'colors' should be declared as a constant because its value is not meant to change. Also, it's a good practice to use uppercase for constant names. For example: 'private static final Color[] COLORS = Color.values();'